airqo-platform/AirQo-api
# Location registry backend

- What does this PR do?
This PR created a location-registry microservice and endpoints to be utilized by the location registry.
- What JIRA task is related to this PR?
[1](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-11)
[2](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-23)
[3](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-26)
[4](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-22)
[5](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-12)
[6](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-24)
[7](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-13)
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/location-registry` Then run `pip install -r requirements.txt` to get all the relevant packages. Afterwards run `flask run` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
partial url: /api/v1/location_registry/create_id
partial url: /api/v1/location_registry/register
partial url: /api/v1/location_registry/locations
partial url: /api/v1/location_registry
partial url: /api/v1/location_registry/edit
partial url: /api/v1/location_registry/update
